k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -750,6 +749,18 @@ private void onDiscard() {
         }
     }
 
+    private void onHighPriority(MenuItem item) {
+        if (isHighPriority) {
+            item.setTitle(getString(R.string.priority_urgent));
+            this.isHighPriority = false;
+            setTitle(getTitle().toString().split(",")[0]);
```

Comment:
Please use a formatting string i.e.:

`setTitle(getString(R.string.high_priority, getTitle().toString().split(",")[0]))`

so that translation can handle it better.